stdlib/3/collections/__init__.pyi

```diff
@@ -285,11 +285,23 @@ class Counter(Dict[_T, int], Generic[_T]):
 
 _OrderedDictT = TypeVar('_OrderedDictT', bound=OrderedDict)
 
+class _OrderedDictKeysView(KeysView[_KT], Reversible[_KT], Generic[_KT]):
```

Comment:
The explicit Generic base class is redundant.

Reply:
As well as `Reversible` IIRC.

Reply:
@ilevkivskyi `Reversible`, or the `def __reversed__(self)` definition?

Reply:
Put differently, is

```
class _OrderedDictKeysView(KeysView[_KT], Reversible[_KT]):
    ...
```

sufficient?

Reply:
I actually wanted to leave the method and remove the base class (because `Reversible`) is a protocol IIRC. But either way is fine if it works.